postgres_composite_types/forms.py

```diff
@@ -146,7 +146,7 @@ def clean(self, value):
                 except forms.ValidationError as error:
                     errors.append(prefix_validation_error(
                         error, code='field_invalid',
-                        prefix='%(label)s: ', params={'label': field.label}))
+                        prefix='%(label)s:', params={'label': field.label}))
```

Comment:
Hmm, I can't explain this space. I think removing it is correct.

Reply:
Now I found out that this is due to a change in Django 2.1:
[Refs #29131](https://github.com/django/django/commit/31ce1f74334add625a4ed82b531d769828237882#diff-2e43aa7479ed400756fabd200389841c)
I have created another commit which will fix it.
[make error prefix compatible with different django versions](https://github.com/danni/django-postgres-composite-types/pull/25/commits/b4cf8f16d1f4d9082b883e5381904a160431e5dd)